StreamDeckPlugin/Events/RegisterForUpdates.cs

```diff
@@ -0,0 +1,17 @@
+﻿using StreamDeckPlugin.Services;
+using System;
+
+namespace StreamDeckPlugin.Events {
+    public class RegisterForUpdates : IEvent {
```

Comment:
This is the code that makes the UI aware that it exists. So it's registering itself so that the UI will send updates to it. Bascially, if this never happens, the streamdeck will never get pushed updates. I have renamed this to "RegisterForUpdatesRequest".